### ISSUE TITLE
Separate templates for Public tenant and all other tenants?

### ISSUE THREAD

**Issue by danielmaddern** [bernardopires/django-tenant-schemas#476]
Is it possible to have 2 sets of templates, 1 for the public tenant and one for every other tenant?

**Comment by bernardopires** [comment]
Yes, you can have different views for the public tenant. See https://django-tenant-schemas.readthedocs.io/en/latest/install.html#tenant-view-routing
